FirebaseRemoteConfig/Sources/RCNConfigContent.m

```diff
@@ -152,6 +152,20 @@ - (void)copyFromDictionary:(NSDictionary *)fromDict
         [dateFormatter setDateFormat:@"yyyy-MM-dd HH:mm:ss"];
         NSString *strValue = [dateFormatter stringFromDate:(NSDate *)value];
         valueData = [(NSString *)strValue dataUsingEncoding:NSUTF8StringEncoding];
+      } else if ([value isKindOfClass:[NSArray class]]) {
+        NSError *error;
+        valueData = [NSJSONSerialization dataWithJSONObject:value options:0 error:&error];
+        if (error) {
+          FIRLogError(kFIRLoggerRemoteConfig, @"I-RCN000075",
```

Comment:
These should be unique. Looks like 76 and 77 are next:

 git grep RCN00007
```
FirebaseRemoteConfig/Sources/RCNConfigDBManager.m:    FIRLogError(kFIRLoggerRemoteConfig, @"I-RCN000075",
FirebaseRemoteConfig/Sources/RCNConfigFetch.m:    FIRLogError(kFIRLoggerRemoteConfig, @"I-RCN000074", @"%@",
FirebaseRemoteConfig/Sources/RCNConfigFetch.m:      FIRLogError(kFIRLoggerRemoteConfig, @"I-RCN000073", @"%@",
FirebaseRemoteConfig/Sources/RCNConfigFetch.m:    FIRLogError(kFIRLoggerRemoteConfig, @"I-RCN000071",
FirebaseRemoteConfig/Sources/RCNConfigSettings.m:      FIRLogNotice(kFIRLoggerRemoteConfig, @"I-RCN000072",
```

Reply:
Gotcha. I saw 75 was the next available number and used it twice. I'll bump one of them to 76.